Formula/livekit.rb

```diff
@@ -0,0 +1,20 @@
+class Livekit < Formula
+  desc "Scalable, high-performance WebRTC SFU. Written in pure Go"
+  homepage "https://livekit.io"
+  url "https://github.com/livekit/livekit/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "acc55775cca1648940706842ace7d453dd71d301689c4f368bf8467ee1460507"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cmd/server" do
+      system "go", "build", "-o", "livekit-server"
+    end
+    bin.install "cmd/server/livekit-server"
+  end
+
+  test do
+    assert_match "livekit-server - High performance WebRTC server", shell_output("#{bin}/livekit-server --help 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the quick review!

Regarding tests: this is a WebRTC server and requires some configuration to get up and running. Would testing for a startup error message be sufficient?

Reply:
@SMillerDev added a test for this, PTAL